app/css/grid.scss

```diff
@@ -0,0 +1,24 @@
+@import '~@openmrs/style-referenceapplication/src/sass/fragments/grid';
+
+.grid-row {
+    @include row();
+}
+
+.button-row {
+    @include row();
+    margin-top: 1%;
+}
+
+@for $i from 1 through 12 {
```

Comment:
It appears you're reimplementing a 12-column grid here. Why is that?

I also noticed the bootstrap scripts were commented out in `app/index.html`. Have you decided to drop bootstrap dependency?

Reply:
At first we had used bootstrap, but then we realised that openmrs has a style guide that we had not used. We decided to drop bootstrap for now and use the [styleguide](https://www.npmjs.com/package/@openmrs/style-referenceapplication). 
> It appears you're reimplementing a 12-column grid here. Why is that?
The style guide has a` grid.sccs` file that defines `mixins` for rows and columns which are just declarations and can not be used directly.

Reply:
Using the *style-guide* does not necessarily exclude the need for bootstrap to provide consistent styling and widgets. Did using bootstrap have any negative effects on the styles?

Reply:
yes, we realised that our UI was not consistent with other interfaces in openmrs, it was a tradeoff for consistency.